config/scheduler/settings.ini

```diff
@@ -446,6 +446,19 @@ pa.scheduler.core.tenant.filter=false
 # It is possible to allow both named domain and empty domain using the syntax pa.scheduler.allowed.domains=,domain1,domain2
 # pa.scheduler.allowed.domains=domain1,domain2
 
+# Regular expression used to control the username format when logging in. Always enabled to prevent injection attacks on 3rd-party authentication like LDAP
+pa.scheduler.username.regexp=^[A-Za-z0-9_\\-@.]+$
+
+# Enable password strength check, used when a user is created or modified
+pa.scheduler.password.strength.enable=false
```

Comment:
I would rather set this to 'true' by default

Reply:
I can't because the default passwords, e.g. admin / admin don't match the requirements